crates/zed/src/zed/only_instance.rs

```diff
@@ -13,14 +15,21 @@ const RECEIVE_TIMEOUT: Duration = Duration::from_millis(35);
 const SEND_TIMEOUT: Duration = Duration::from_millis(20);
 
 fn address() -> SocketAddr {
+    // These port numbers have a 10 port offset between each release channel
+    // to avoid conflicts between different users. This means that only up to
+    // 10 users can run Zed on the same machine at the same time.
     let port = match *release_channel::RELEASE_CHANNEL {
         ReleaseChannel::Dev => 43737,
-        ReleaseChannel::Preview => 43738,
-        ReleaseChannel::Stable => 43739,
-        ReleaseChannel::Nightly => 43740,
+        ReleaseChannel::Preview => 43747,
+        ReleaseChannel::Stable => 43757,
+        ReleaseChannel::Nightly => 43767,
     };
+    let user_id = Uid::current().as_raw() as u16;
```

Comment:
Are user UIDs guaranteed to be consecutive?

Reply:
Good point, they are definitely not always consecutive. There might be gaps so it's safer to have a bigger buffer in between ports for each  release channel.